tests/test_build_wheel.py

```diff
@@ -19,7 +21,11 @@ def test_build_wheel(addon1_with_pyproject: Path, tmp_path: Path) -> None:
             assert "odoo/addons/addon1/pyproject.toml" not in names
 
 
-def test_build_wheel_without_scm(tmp_path: Path) -> None:
+def test_build_wheel_without_scm(
+    tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+) -> None:
+    # Make sure we can't find git executable
+    monkeypatch.setenv("PATH", "/foo")
```

Comment:
Don't we need another test for this specific case?

Reply:
This test is named `test_build_wheel_without_scm`. So, it seems to me like the appropriate place to test that you can build without Git.

However, as you can see, when removing Git from its $PATH, the test actually fails. This means that you need Git installed, even if you're not gonna use it.

Reply:
Yes, but we want to test two cases: when `git` is absent, and when we are not in a git repo. These are two different test cases, no?